docs/asset-store/docs/08-01-gateway-mode-set-up.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Set Minio to the Google Cloud Storage (GCS) Gateway mode
+type: Tutorials
+---
+
+This tutorial shows how to set Minio to the Google Cloud Storage (GCS) Gateway mode. Follow these guidelines to create a secret and update the Asset Store deployment.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl)
+- [helm](https://github.com/helm/helm#install)
+- [Google Cloud Platform (GCP)](https://cloud.google.com)  access
+
+## Steps
+
+To set Minio to a Gateway mode, you need a Secret with a service account that has the **Storage Admin** role permissions.
+
+### Create a Secret
+
+1. Open the [service accounts page](https://console.cloud.google.com/iam-admin/serviceaccounts).
+2. Select one of the projects or create a new one. Note down the project ID as you must use it to update the Asset Store deployment.
+3. Click **Create service account**, name your account, and click **Create**.
+4. Set the `Storage Admin` role.
+5. Click **Create key** and choose `JSON` as a key type.
+6. Save the `JSON` file.
+7. Create a Secret from the JSON file by running this command:
+    ```bahs
+    kubectl create secret generic assetstore-gcs-credentials --from-file=service-account.json={filename} --namespace kyma-system
+    ```
+
+### Update the Asset Store deployment
+
+To update the Asset Store deployment, run this command:
+
+```bash
+helm upgrade assetstore resources/assetstore --namespace kyma-system --wait=true --reuse-values --set minio.persistence.enabled=false --set minio.gcsgateway.enabled=true --set minio.gcsgateway.replicas=1 --set minio.gcsgateway.gcsKeySecret=assetstore-gcs-credentials --set minio.gcsgateway.projectId={gcp-project} --set minio.defaultBucket.enabled=false
+```
+
+>**NOTE:** This is an alpha version of the Asset Store. In this version, the GCS `content` bucket is not available in the Minio Gateway mode.
```

Comment:
not sure what is the purpose of this note. We know about legacy issue with have with `content` bucket as we should simply not release this docs until it is solved, otherwise customer will break his functionality with this steps